openedx/features/course_search/urls.py

```diff
@@ -4,7 +4,7 @@
 
 from django.conf.urls import url
 
-from views.course_search import CourseSearchView, CourseSearchFragmentView
+from views.course_search import CourseSearchFragmentView, CourseSearchView
```

Comment:
Oh, I see. If you otherwise are not touching the file, I would revert this change.